### DISCUSSION TITLE
article/read-book-with-chatgpt-get-together

### DISCUSSION THREAD

**Discussion by giscus** [lycfyi/NotionNext#14] [General]
# article/read-book-with-chatgpt-get-together

这本书名为"Get Together"，由People & Company出版，是一本关于如何建立和维护社区的指南。该书提供了关于培养一群热情人群的每个阶段的故事、提示和原则。作者强调了与人一起建立社区，而不是为他们建立社区的重要性。书中突出了每个组织建立和维护一个繁荣社区的潜力。

https://lyc.fyi/article/read-book-with-chatgpt-get-together

**Comment by goto911** [comment]
Yucheng，你的rss订阅，文章不全，检查一下呢。谢谢

**Reply by lycfyi** [reply]
感谢goto911反馈！可能是因为我有时候会在第一次发布后还更新内容，而后面更新的内容不会被RSS捡起来。我测试一下是不是，是的话我下次争取第一次就把内容尽量完整放出来 😅
